File: curso c++/ejemplos/problema13.cpp
//dado un caracter determinar si es vocal
#include <iostream>
using namespace std;
int main (){
    char caracter;
    string respuesta;
    cout << "ingrese el caracter: ";
    cin >> caracter;

    
    if (caracter == 'a' || caracter== 'A')
        respuesta = "ES VOCAL";
    if (caracter == 'e' || caracter == 'E')
        respuesta = "ES VOCAL";
    if (caracter == 'i' || caracter == 'I')
        respuesta = "ES VOCAL";
    if (caracter == 'o' || caracter == 'O')
        respuesta = "ES VOCAL";
    if (caracter == 'u' || caracter == 'U')
        respuesta = "ES VOCAL";
    respuesta= "NO ES VOCAL ";
    cout << "el caracter ingresado es: "<< respuesta;

    return 0;
}

File: curso python/ejemplos python/problema13.py
#dado un caracter determinar si es vocal o no CON FUNCION


def vocal(c):
    resp="no es vocal"
    if c == 'a' or c == 'A':
        resp= "es vocal"
    if c == 'e' or c == 'E':
        resp= "es vocal"
    if c == 'i' or c == 'I':
        resp= "es vocal"
    if c == 'o' or  c == 'O':
        resp= "es vocal"
    if c == 'u' or c == 'U':
        resp= "es vocal"
    return resp
c=input('ingrese el caracter: ')
print (vocal(c))